### PR TITLE
Initial work on getting samba integrated into the system. 

### PR DESCRIPTION
**Pull request by timodwhit**
Update also adds readme about mounting the drives.

### CODE REVIEW

File: Berksfile.lock [hunk 1]
```diff
@@ -4,7 +4,7 @@ DEPENDENCIES
   dmg (~> 2.0.8)
   drupal
     git: https://github.com/vampd/drupal
-    revision: 476551cb0702a7df6ae19a23c92642174be1ea47
+    revision: 064b1beeb5919a3fd8fbe4b374c24745f3e9b411
```
Comment (arknoll) [inline Berksfile.lock]: does this need to be updated again?

Comment (timodwhit) [inline Berksfile.lock]: Most likely

**Comment by arknoll** [comment]
testing

**Comment by arknoll** [comment]
test passed. Can you re-roll the berksfile.lock?

**Pull request merged**